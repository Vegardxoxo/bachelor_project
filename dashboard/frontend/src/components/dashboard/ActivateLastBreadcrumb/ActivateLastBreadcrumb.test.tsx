import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';
import ActiveLastBreadcrumb from "./ActivateLastBreadcrumb";
import {render} from '@testing-library/react';

describe('ActiveLastBreadcrumb', () => {

    it('renders without crashing', () => {
    render(
      <MemoryRouter initialEntries={['/']}>
        <ActiveLastBreadcrumb />
      </MemoryRouter>
    );
  });
    it('matches snapshot', async () => {
        const testRenderer = renderer.create(
            <MemoryRouter initialEntries={['/']}>
                <ActiveLastBreadcrumb/>
            </MemoryRouter>
        );
        expect(testRenderer.toJSON()).toMatchSnapshot();
    });
});
