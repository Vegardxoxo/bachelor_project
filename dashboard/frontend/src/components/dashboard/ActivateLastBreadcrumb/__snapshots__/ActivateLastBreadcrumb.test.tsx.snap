// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActiveLastBreadcrumb matches snapshot 1`] = `
<nav
  aria-label="breadcrumb"
  className="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-63vwl1-MuiTypography-root-MuiBreadcrumbs-root"
  id="breadcrumbs"
>
  <ol
    className="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
  >
    <li
      className="MuiBreadcrumbs-li"
    >
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-le90w3-MuiTypography-root-MuiLink-root"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
      >
        Dashboard
      </a>
    </li>
  </ol>
</nav>
`;
